src/compo/viirs_biaswriter.py

```diff
@@ -19,25 +19,18 @@
     type=str, required=True)
 
 args = parser.parse_args()
-predictor_out = netCDF4.stringtochar(np.array(['constant'], 'S8'))
 channels_out = [4]
 ncfile = netCDF4.Dataset(args.output, mode='w', format='NETCDF4')
-coef_dim = ncfile.createDimension('bias_coefficients', 1)
-coef_err_dim = ncfile.createDimension('bias_coeff_errors', 1)
-npredictors = ncfile.createDimension('npredictors', 1)
-nchannels = ncfile.createDimension('nchannels', 1)
-coef = ncfile.createVariable('bias_coefficients', np.float, ('bias_coefficients', ))
-coef_error = ncfile.createVariable('bias_coeff_errors', np.float, ('bias_coeff_errors', ))
-predictors = ncfile.createVariable('predictors', 'S8', ('npredictors', ))
-channels = ncfile.createVariable('channels', np.int, ('nchannels', ))
-coef.units = 'Aerosol optical depth'
-coef.long_name = 'bias_coefficients'
```

Comment:
This brings a question to me whether we would be able to tell what variables these coefficients are for. It seems we are not leaving any trace in the coefficient files. Is it true?

Reply:
There is a problem in that AOD and brightnessTemperature are ambiguous, we can know channel or variable but not both. Is that a problem? The obs space would never have Tb and AOD both would it?